drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -1058,6 +1039,7 @@ static int bnxt_init_board(struct rte_eth_dev *eth_dev)
 	return rc;
 }
 
+#define ALLOW_FUNC(x)	bp->pf.vf_req_fwd[((x)>>5)] &= ~rte_cpu_to_le_32((1<<((x) & 0x1f)))
```

Comment:
Do we have any MACROs from HWRM which we could instead of these hardcoded values?